### ISSUE TITLE
2017 ...

### ISSUE THREAD

**Issue by mikejnpc** [brichard19/BitCrack#3]
does this run in 2017 VS? i see the errors being thrown for some of the sections... 
had issues looking for 2015  also  cuda 8 is not supported in vs2017 -- any help on this appreciated

**Comment by mikejnpc** [comment]
Okay sounds good. Ummm is it all automated in regards to when it corrlates a matching key... ?? i am running it on a x64 so started it from a different path...
Addionally i am unsure why v 0.0.4 worked as opposed to the more recent version where when loaded into VS would throw errors.
I assume its all cmd entry and loaded thru the keyfinder.exe not located in the root dir...
Other than that im good and appriciate you emailing me....
have you ever had a positive result as i am trying to recover my old wallet...nothing naferous ... just wondering if its ever worked which assume does on earlier wallets than after 2012 when mulisig came to be supported.
Its a nice build ..i respect your time into itThx
Mike




Sent via the Samsung Galaxy S8 Active, an AT&T 4G LTE smartphone
-------- Original message --------From: brichard19 <notifications@github.com> Date: 5/22/18  6:21 PM  (GMT-08:00) To: brichard19/BitCrack <BitCrack@noreply.github.com> Cc: mikejnpc <mikejnpc@gmail.com>, Author <author@noreply.github.com> Subject: Re: [brichard19/BitCrack] 2017 ... (#3) 
Hi there,
For now this project is using VS2015 with CUDA 8.
Thanks

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
{"@context":"http://schema.org","@type":"EmailMessage","potentialAction":{"@type":"ViewAction","target":"https://github.com/brichard19/BitCrack/issues/3#issuecomment-391190347","url":"https://github.com/brichard19/BitCrack/issues/3#issuecomment-391190347","name":"View Issue"},"description":"View this Issue on GitHub","publisher":{"@type":"Organization","name":"GitHub","url":"https://github.com"}}
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/brichard19/BitCrack","title":"brichard19/BitCrack","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/brichard19/BitCrack"}},"updates":{"snippets":[{"icon":"PERSON","message":"@brichard19 in #3: Hi there,\r\n\r\nFor now this project is using VS2015 with CUDA 8. \r\n\r\nThanks"}],"action":{"name":"View Issue","url":"https://github.com/brichard19/BitCrack/issues/3#issuecomment-391190347"}}}
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "37567f93-e2a7-4e2a-ad37-a9160fc62647",
"title": "Re: [brichard19/BitCrack] 2017 ... (#3)",
"sections": [
{
"text": "",
"activityTitle": "**brichard19**",
"activityImage": "https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png",
"activitySubtitle": "@brichard19",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"brichard19/BitCrack\",\n\"issueId\": 3,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close issue",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueClose\",\n\"repositoryFullName\": \"brichard19/BitCrack\",\n\"issueId\": 3\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/brichard19/BitCrack/issues/3#issuecomment-391190347"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 334556975\n}"
}
],
"themeColor": "26292E"
}

**Comment by mikejnpc** [comment]
if you care to assist i wouldnt mind donating 1/3rd to your project
efforts.
everyone needs to eat ... :)

think it over.. cheers!

**Comment by brichard19** [comment]
Hi,

Can you describe the problem you are having with loading the solution in VS? The project loads and builds fine on my system.


In regards to your second question, the tool does have positive results. For instance, the address 12bahMGWAnSEYS3PKURvhScm8GmZhejY7t can be solved in under a minute.

I have been testing against addresses I've generated using this tool (by specifying the secret exponent): [https://brainwalletx.github.io/#generator](https://brainwalletx.github.io/#generator). The AddrGen.exe tool produces the same output too.

I'm afraid you will not be able to recover your old wallet with this tool, though. This is a brute-force tool designed to assist in solving the Bitcoin puzzle transactions:  [https://blockchain.info/tx/08389f34c98c606322740c0be6a7125d9860bb8d5cb182c02f98461e5fa6cd15
](https://blockchain.info/tx/08389f34c98c606322740c0be6a7125d9860bb8d5cb182c02f98461e5fa6cd15
). More details here: [https://bitcointalk.org/index.php?topic=1306983.0](https://bitcointalk.org/index.php?topic=1306983.0).

These private keys are in known ranges, and the most efficient method is brute force.


Was your wallet password protected? It may be feasible to add in a password cracking feature.

**Comment by mikejnpc** [comment]
richard, could you send me a email directly or jabber works too... same
nickname @yax.im
just dont want a bunch of stuff posted in a way where it inst controlled :)
thx for being so attentive and can discuss my problem and possible
solution,...

**Comment by adminnu** [comment]
I today set up cuda 9.2 and vs 2017 in an hour. Everything was compiled without problems and works.

**Comment by brichard19** [comment]
Hi,

That's great news! I'll look to transitioning to VS2017 and CUDA 9.

Thanks

**Comment by natedawg469** [comment]
Could we implement a timer into the initialization process? I have been trying so many different combinations of the parameters to try get the program to start faster and I can't figure it all out. It just says initialization... and sometimes after 12 hours its still saying that.

**Comment by adminnu** [comment]
> Hi,
> 
> That's great news! I'll look to transitioning to VS2017 and CUDA 9.
> 
> Thanks
> 
Yes, everything is pretty simple, just change the properties of projects to 9.2 and obj everywhere.
And the truth is I did not put the 2015 version, I changed the permissible version in host_config.h
It was enough to make it work.

**Comment by natedawg469** [comment]
@adminnu Did it make the program run any better?
